external-crates/move/crates/move-cli/tests/build_tests/simple_new/args.exp

```diff
@@ -3,14 +3,74 @@ External Command `cat P1/Move.toml`:
 [package]
 name = "P1"
 
+# version = "0.1.1"  # semver "maj.min.patch"
+# license = ""       # e.g., "MIT", "GPL", "Apache 2.0"
+# authors = ["..."]  # e.g., ["Joe Smith (joesmith@noemail.com)", "John Snow (johnsnow@noemail.com)"]
+
 [dependencies]
 
+# For remote import, use the `{ git = "...", subdir = "...", rev = "..." }`.
+# Revision can be a branch, a tag, and a commit hash.
+# MyRemotePackage = { git = "https://some.remote/host.git", subdir = "remote/path", rev = "main" }
+
+# For local dependencies use `local = path`. Path is relative to the package root
+# Local = { local = "../path/to" }
+
+# To resolve a version conflict and force a specific version for dependency
+# override use `override = true`
+# Override = { override = true, local = "../conflicting/version" }
+
 [addresses]
+
+# Named addresses will be accessible in Move as `@name`. They're also exported:
+# for example, `std = "0x1"` is exported by the Standard Library.
+# alice = "0xA11CE"
+
+[dev-dependencies]
+# The dev-dependencies section allows overriding dependencies for `--test` and
+# `--dev` modes. New dependencies cannot be introduced
+# Local = { local = "../path/to/dev-build" }
+
+[dev-addresses]
+# The dev-addresses section allows overloading named addresses for the `--test`
```

Comment:
"overloading" seems like the wrong word here -- would maybe "assign a specific value to" or "overwrite" be more accurate here?

Reply:
Thanks! Patched!